web/pdf_page_view.js

```diff
@@ -582,6 +603,50 @@ var PDFPageView = (function PDFPageViewClosure() {
 
       return result;
     },
+
+    paintOnSvg: function PDFPageView_paintOnSvg(canvasWrapper) {
+      if (typeof PDFJSDev !== 'undefined' &&
+          PDFJSDev.test('FIREFOX || MOZCENTRAL || CHROME')) {
+         return Promise.resolve('SVG rendering is not supported.');
+      }
+
+      var cancelled = false;
+      var ensureNotCancelled = function () {
+        if (cancelled) {
+          throw 'cancelled';
+        }
+      };
+
+      var self = this;
+      var pdfPage = this.pdfPage;
+      var SVGGraphics = pdfjsLib.SVGGraphics;
+      var viewport = this.viewport;
```

Comment:
This intermediate variable is not really used. Let's remove it and use `this.viewport` in the line below.